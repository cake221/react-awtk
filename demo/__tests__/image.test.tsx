import "../image"

describe("test button", ()=>{

  it("toMatchSnapshot", ()=>{
    expect(__AwtkSnapshot).toMatchSnapshot()
  })
})
