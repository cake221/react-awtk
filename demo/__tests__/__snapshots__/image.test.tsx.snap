// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test button toMatchSnapshot 1`] = `
Array [
  "getRootHostContext",
  "getChildHostContext",
  "shouldSetTextContent",
  "getChildHostContext",
  "shouldSetTextContent",
  "createInstance",
  "funtion  image_create( parent, x, y, w, h ) :null, 0, 0, 0, 0",
  "funtion  widget_use_style( widget, style ) :undefined, border",
  "funtion  image_set_draw_type( widget, draw_type ) :undefined, IMAGE_DRAW_ICON",
  "funtion  image_base_set_image( widget, name ) :undefined, earth",
  "funtion  image_base_set_rotation( widget, rotation ) :undefined, 0.5",
  "funtion  image_base_set_scale( widget, scale_x, scale_y ) :undefined, 1, 2",
  "finalizeInitialChildren",
  "getChildHostContext",
  "shouldSetTextContent",
  "createInstance",
  "funtion  image_create( parent, x, y, w, h ) :null, 0, 0, 0, 0",
  "funtion  widget_use_style( widget, style ) :undefined, border",
  "funtion  image_set_draw_type( widget, draw_type ) :undefined, IMAGE_DRAW_CENTER",
  "funtion  image_base_set_image( widget, name ) :undefined, earth",
  "finalizeInitialChildren",
  "getChildHostContext",
  "shouldSetTextContent",
  "createInstance",
  "funtion  image_create( parent, x, y, w, h ) :null, 0, 0, 0, 0",
  "funtion  widget_use_style( widget, style ) :undefined, border",
  "funtion  image_set_draw_type( widget, draw_type ) :undefined, IMAGE_DRAW_SCALE_AUTO",
  "funtion  image_base_set_image( widget, name ) :undefined, earth",
  "finalizeInitialChildren",
  "getChildHostContext",
  "shouldSetTextContent",
  "createInstance",
  "funtion  image_create( parent, x, y, w, h ) :null, 0, 0, 0, 0",
  "funtion  widget_use_style( widget, style ) :undefined, border",
  "funtion  image_set_draw_type( widget, draw_type ) :undefined, IMAGE_DRAW_SCALE",
  "funtion  image_base_set_image( widget, name ) :undefined, 1",
  "finalizeInitialChildren",
  "getChildHostContext",
  "shouldSetTextContent",
  "createInstance",
  "funtion  image_create( parent, x, y, w, h ) :null, 0, 0, 0, 0",
  "funtion  widget_use_style( widget, style ) :undefined, border",
  "funtion  image_set_draw_type( widget, draw_type ) :undefined, IMAGE_DRAW_SCALE_W",
  "funtion  image_base_set_image( widget, name ) :undefined, 1",
  "finalizeInitialChildren",
  "getChildHostContext",
  "shouldSetTextContent",
  "createInstance",
  "funtion  image_create( parent, x, y, w, h ) :null, 0, 0, 0, 0",
  "funtion  widget_use_style( widget, style ) :undefined, border",
  "funtion  image_set_draw_type( widget, draw_type ) :undefined, IMAGE_DRAW_SCALE_H",
  "funtion  image_base_set_image( widget, name ) :undefined, 1",
  "finalizeInitialChildren",
  "getChildHostContext",
  "shouldSetTextContent",
  "createInstance",
  "funtion  image_create( parent, x, y, w, h ) :null, 0, 0, 0, 0",
  "funtion  widget_use_style( widget, style ) :undefined, border",
  "funtion  image_set_draw_type( widget, draw_type ) :undefined, IMAGE_DRAW_REPEAT",
  "funtion  image_base_set_image( widget, name ) :undefined, bricks",
  "finalizeInitialChildren",
  "getChildHostContext",
  "shouldSetTextContent",
  "createInstance",
  "funtion  image_create( parent, x, y, w, h ) :null, 0, 0, 0, 0",
  "funtion  widget_use_style( widget, style ) :undefined, border",
  "funtion  image_set_draw_type( widget, draw_type ) :undefined, IMAGE_DRAW_REPEAT_X",
  "funtion  image_base_set_image( widget, name ) :undefined, bricks",
  "finalizeInitialChildren",
  "getChildHostContext",
  "shouldSetTextContent",
  "createInstance",
  "funtion  image_create( parent, x, y, w, h ) :null, 0, 0, 0, 0",
  "funtion  widget_use_style( widget, style ) :undefined, border",
  "funtion  image_set_draw_type( widget, draw_type ) :undefined, IMAGE_DRAW_REPEAT_Y",
  "funtion  image_base_set_image( widget, name ) :undefined, bricks",
  "finalizeInitialChildren",
  "createInstance",
  "funtion  window_create( parent, x, y, w, h ) :null, 0, 0, 0, 0",
  "funtion  widget_set_children_layout( widget, params ) :undefined, rows:3 cols:3 margin:2 spacing:2",
  "appendInitialChild",
  "appendInitialChild",
  "appendInitialChild",
  "appendInitialChild",
  "appendInitialChild",
  "appendInitialChild",
  "appendInitialChild",
  "appendInitialChild",
  "appendInitialChild",
  "finalizeInitialChildren",
  "prepareForCommit",
  "appendNodeChild",
  "resetAfterCommit",
  "getPublicInstance",
  "funtion  widget_add_child( widget, child ) :undefined, [object Object]",
  "funtion  widget_add_child( widget, child ) :undefined, [object Object]",
  "funtion  widget_add_child( widget, child ) :undefined, [object Object]",
  "funtion  widget_add_child( widget, child ) :undefined, [object Object]",
  "funtion  widget_add_child( widget, child ) :undefined, [object Object]",
  "funtion  widget_add_child( widget, child ) :undefined, [object Object]",
  "funtion  widget_add_child( widget, child ) :undefined, [object Object]",
  "funtion  widget_add_child( widget, child ) :undefined, [object Object]",
  "funtion  widget_add_child( widget, child ) :undefined, [object Object]",
  "funtion  widget_layout( widget ) :undefined",
  "win1 子控件 数量: 9",
  "funtion  tk_get_pointer_x(  ) :",
  "funtion  tk_get_pointer_y(  ) :",
  "funtion  tk_is_pointer_pressed(  ) :",
]
`;
